lib/versioning/VersionID.js

```diff
@@ -8,45 +8,52 @@
 
 // the lengths of the components in bytes
 const LENGTH_TS = 14; // timestamp: epoch in ms
-const LENGTH_SQ = 6; // position in ms slot
-const LENGTH_ST = 5; // site identifier
+const LENGTH_SEQ = 6; // position in ms slot
+const LENGTH_SITE = 5; // site identifier
 
 // empty string template for the variables in a versionId
 const TEMPLATE_TS = new Array(LENGTH_TS + 1).join('0');
-const TEMPLATE_SQ = new Array(LENGTH_SQ + 1).join('0');
-const TEMPLATE_ST = new Array(LENGTH_ST + 1).join(' ');
+const TEMPLATE_SEQ = new Array(LENGTH_SEQ + 1).join('0');
+const TEMPLATE_SITE = new Array(LENGTH_SITE + 1).join(' ');
 
 /**
- * Padding a string representation of a value with a given template.
- * For example: pad('foo', '00000') equals '00foo' for left padding
- * and 'foo00' for right padding.
+ * Left-pad a string representation of a value with a given template.
+ * For example: pad('foo', '00000') gives '00foo'.
  *
  * @param {any} value - value to pad
  * @param {string} template - padding template
- * @param {boolean} [right] - padding right (default is padding left)
  * @return {string} - padded string
  */
-function pad(value, template, right = false) {
-    if (right) {
-        return `${value}${template}`.slice(0, template.length);
-    }
+function padLeft(value, template) {
     return `${template}${value}`.slice(-template.length);
 }
 
+/**
+ * Right-pad a string representation of a value with a given template.
+ * For example: pad('foo', '00000') gives 'foo00'.
+ *
+ * @param {any} value - value to pad
+ * @param {string} template - padding template
+ * @return {string} - padded string
+ */
+function padRight(value, template) {
+    return `${value}${template}`.slice(0, template.length);
+}
+
 // site identifier, like PARIS, TOKYO; will be trimmed if exceeding max length
-const SITE_ID = pad(process.env.SITE_ID, TEMPLATE_ST, true);
-// const SITE_ID = `${process.env.SITE_ID}${TEMPLATE_ST}`.slice(0, LENGTH_ST);
+const SITE_ID = padRight(process.env.SITE_ID, TEMPLATE_SITE);
```

Comment:
this should be 'replication_group_token'. It would be good to have some default beyond undefined and some explanation in the s3 readme for open source users.  Instead of an env variable, could we use the s3 config and pass an argument when calling the generateVersionId function?